pkg/csi/service/wcp/controller.go

```diff
@@ -411,6 +411,11 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 			return nil, csifault.CSIUnimplementedFault, logger.LogNewErrorCodef(log, codes.Unimplemented,
 				"support for topology requirement with both zone and hostname labels is not yet implemented.")
 		} else if zoneLabelPresent {
+			// StorageTopologyType should be set if topology label is present
+			if storageTopologyType == "" {
+				return nil, csifault.CSIInternalFault, logger.LogNewErrorCode(log, codes.Internal,
```

Comment:
Should we return `InvalidArgument` instead of `Internal` error?

Reply:
Good point.